cmd/web-handlers.go

```diff
@@ -588,7 +645,8 @@ func (web *webAPIHandlers) SetAuth(r *http.Request, args *SetAuthArgs, reply *Se
 			reply.PeerErrMsgs[host.String()] = err.Error()
 		}
 	} else {
-		reply.Token = newAuthToken()
+		reply.Token, err = authenticateWeb(creds.AccessKey, creds.SecretKey)
+		logger.CriticalIf(context.Background(), err)
```

Comment:
Adding `newContext` in the beginning of the handler would be good, instead of `context.Background()`

Reply:
okay @kannappanr